06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+require_relative './last_frame'
+require_relative './shot'
+
+class Game
+  def initialize
+    @frames = Array.new(10) do |index|
+      index == 9 ? LastFrame.new : Frame.new
+    end
+  end
+
+  def play(marks)
+    index = 0
+    @frames.each do |frame|
+      loop do
+        break if frame.full?
+
+        shot = Shot.new(marks[index])
+        frame.set(shot)
+        index += 1
+      end
+    end
+
+    sum_scores(@frames)
```

Comment:
前回のコメントの意図がうまく伝わってなかったかも。

```suggestion
    @frames.sum do |frame|
      frame.calc_score(@frames)
    end
```

のような実装をイメージしていました。

なお、indexは `Frame.new(index)`みたいな感じで、newするタイミングでセットする想定です